internal/querynode/task.go

```diff
@@ -265,27 +271,21 @@ func (w *watchDmChannelsTask) Execute(ctx context.Context) error {
 			return err
 		}
 	}
-	var l loadType
-	if loadPartition {
-		l = loadTypePartition
-	} else {
-		l = loadTypeCollection
-	}
 	sCol, err := w.node.streaming.replica.getCollectionByID(collectionID)
 	if err != nil {
 		return err
```

Comment:
This can be judged before entering the task queue, and if error, directly returns

Reply:
collection is created in task.go before get, this error would not happen logically